salt/docs/init.sls

```diff
@@ -78,22 +78,28 @@ docsbuild-sentry:
     - name: SENTRY_DSN
     - value: {{ pillar.get('docs', {}).get('sentry', {}).get('dsn', '') }}
 
-docsbuild-full:
+docsbuild-no-html:
   cron.present:
-    - identifier: docsbuild-full
-    - name: /srv/docsbuild/venv/bin/python /srv/docsbuild/scripts/build_docs.py
+    - identifier: docsbuild-no-html
+    - name: >
+        /srv/docsbuild/venv/bin/python
+        /srv/docsbuild/scripts/build_docs.py
+        --select-output=no-html
     - user: docsbuild
     - minute: 7
+    - hour: 6
     - require:
       - cmd: virtualenv-dependencies
 
-docsbuild-quick:
-  cron.absent:
-    - identifier: docsbuild-quick
-    - name: /srv/docsbuild/venv/bin/python /srv/docsbuild/scripts/build_docs.py -q
+docsbuild-only-html:
+  cron.present:
+    - identifier: docsbuild-only-html
+    - name: >
+        /srv/docsbuild/venv/bin/python
+        /srv/docsbuild/scripts/build_docs.py
+        --select-output=only-html
     - user: docsbuild
-    - minute: 7
-    - hour: 2-23/3
+    - minute: 16
```

Comment:
Ah, I didn't realise. Would the below work as a dummy?
```suggestion
    - minute: 16

docsbuild-full:
  cron.absent:
    - identifier: docsbuild-full
    - name: echo

    - minute: 42
```

Reply:
I am mobile at the moment, but I think it may be as simple as ensuring `cron.absent` is passed the `user` and `identifier` of the old job.

Reply:
Added, sorry for the force push.